src/DacpacTool/Microsoft.SqlTools.ManagedBatchParser/version.txt

```diff
@@ -1 +1 @@
-5.0.20250221.1
+5.0.20250220.4
```

Comment:
Hmm, this change makes me wonder if the automation made some kind of mistake. The version actually appears to be a day _older_ than the previous. When I look at [the sqltoolsservice releases](https://github.com/microsoft/sqltoolsservice/releases), it appears `5.0.20250221.1` is marked as "pre-release", and `5.0.20250220.4` is marked as "latest".
![image](https://github.com/user-attachments/assets/98ceb126-1f2d-403e-a779-590ce6b30a87)

Reply:
I'd note that when I curl the `/releases/latest` endpoint (https://api.github.com/repos/microsoft/sqltoolsservice/releases/latest), `5.0.20250220.4` is what gets returned. So I think _this_ change is correct, but I'm not sure what happened last week. If I had to guess, I'd guess that `5.0.20250221.1` wasn't properly marked as pre-release. But I'm going to keep an eye on this for the next few weeks and see if any adjustments are needed.